pkg/dogstatsd/server.go

```diff
@@ -501,6 +502,16 @@ func findOriginTags(origin string) []string {
 			tags = append(tags, originTags...)
 		}
 	}
+
+	// Include orchestrator scope tags if the cardinality is set to orchestrator
```

Comment:
Nit: should we really do that in `findOriginTags`? We're not testing for `listeners.NoOrigin` and it not seems that much related (I may be wrong on the second part). Feel free to ignore.

Reply:
So `findOriginTags` carries the notion of get tags for the case of "no entity id provided". Not sure if @ahmed-mez or @clamoriniere have an opinion on this?

Reply:
yes technically we're not adding origin tags here 🤔